### PR TITLE
Tickets/dm 34594

### PR DESCRIPTION
**Pull request by womullan**
start focus times

### CODE REVIEW

File: team/focus-times.rst [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
 
 DM will observe  **Focus Friday**. 
 
+DM will observe  **Focus Weeks** around the end/start of year and in Summer and Autumn.
+
+Focus Weeks 2022
+================
+The DM Focus weeks  will be
+- June 13-17 
+- September 18-22
```
Comment (isullivan) [inline team/focus-times.rst:14]: September 19-23?

File: communications/slack-culture.rst [hunk 2]
```diff
@@ -320,7 +320,7 @@ This can be especially difficult if someone starts a thread off an older message
 Focus Friday
```
Comment (ktlim) [inline communications/slack-culture.rst:320]: Adjust this heading too.

**Review by frossie — CHANGES_REQUESTED** [review]
So I obviously don't mind, but I think we had conceived of the "meeting holidays" to not be under Focus Friday rules, and instead (1) cancel all standing meetings (2) highlight to staff that it's a good time to take time off without "falling behind". This was quite a lot laxer than Focus Friday rules. We should probably have a wider discussion to make sure we agree on the specifics?

**Comment by womullan** [comment]
ok on agenda for the morning

**Pull request closed without merging**